float64[] point1 # start position (m)
float64[] point2 # end position (m)
float64 linear_vel # maximum linear velocity (m/s)
float64 linear_acc # maximum linear acceleration (m/s^2)
---
bool result
